src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -786,6 +787,7 @@ public bool IsTypeDesc
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static TypeHandle TypeHandleOf<T>()
+            where T : allows ref struct
```

Comment:
Nit: This is internal runtime API. This change does not need to be in this PR.

 We can add the `allows ref struct` to the internal runtime APIs once needed. There are number of them.

Reply:
Yup, I'm not sure how that ended up adding that here; it wasn't intentional. Will remove.